src/web/js/trove/google-charts.js

```diff
@@ -0,0 +1,226 @@
+(function(){var a="\n//# sourceURL=",k="' of type ",n='<script type="text/javascript" src="',p="SCRIPT",r="array",t="complete",u="function",v="google.charts.load",w="hasOwnProperty",x="number",y="object",z="pre-45",A="propertyIsEnumerable",B="string",C="text/javascript",D="toLocaleString";function E(){return function(b){return b}}function F(){return function(){}}function G(b){return function(){return this[b]}}var I,J=J||{};J.scope={};
```

Comment:
Yikes -- do we need to commit this to our repo?  Is there an npm package we could use instead?

Reply:
My bad! This can be gone. I include it in `editor.html` instead.

Reply:
If it's just import statements, I wonder if we could make `npm-dependencies` do the right thing. That runs through `browserify`, which should be able to handle import statements. I wonder if that file could have an `import` statement in it without breaking anything.